apps/dashboard/dashboard-backend-service.yaml

```diff
@@ -0,0 +1,15 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: dashboard-backend
+  namespace: nsm-system
+spec:
+  selector:
+    app: dashboard
+  ports:
+    - name: http
+      protocol: TCP
+      port: 3001
+      targetPort: 3001
+  clusterIP: None
```

Comment:
`clusterIP: None` means this will be a headless service: k8s will not allocate a special IP for the service, and instead will return direct pod IP for DNS queries.
Reference: https://kubernetes.io/docs/concepts/services-networking/service/#headless-services

I mean, it will not hurt, but it will not help either. It doesn't help security in any way.

Reply:
I feel like this line implies that this service _needs to be_ headless, while in reality it doesn't.
I think that this is bad for an official example.

Reply:
changed the service type to ClusterIP